api/client/service.proto

```diff
@@ -45,3 +46,34 @@ service NighthawkService {
   rpc ExecutionStream(stream ExecutionRequest) returns (stream ExecutionResponse) {
   }
 }
+
+// Encapsulates an ExecutionResponse.
+message StoreExecutionRequest {
+  // Response contains the effective execution id, which will serve as the lookup key.
+  ExecutionResponse execution_response = 1;
+}
+
+// Empty return value message, that serves as a future extension point.
+message StoreExecutionResponse {
+}
+
+message SinkRequest {
+  // Unique id for lookup purposes.
```

Comment:
Assuming this is required, can we clearly indicate that it is?

```
// Unique id for lookup purposes. Required.
```

Alternatively if it isn't, can we describe the behavior when an empty string is provided?

Reply:
Done: d89620e32d3b28caf5fd1eab11b8bff794bdcbca